UpdateDependencies.ps1

```diff
@@ -16,7 +16,7 @@ param(
     [string]$GitHubProject='corefx',
     [string]$GitHubUpstreamBranch='master',
     # a semi-colon delimited list of GitHub users to notify on the PR
-    [string]$GitHubPullRequestNotifications='')
+    [string]$GitHubPullRequestNotifications='stephentoub;ericstj;ellismg;weshaggard;joshfree')
```

Comment:
It's coming. I need to work with the leads and PMs on where exactly this information should go long term. It's just been a bad 2 weeks with //build and ask mode to get something like that decided.

The thought is more information than just the build versions could go in this repo. We could use it to keep track of any cross-repo information.

There are a few ideas on where it could live. One is maybe in a folder under dotnet/Home. Or a new repo, but not sure on the naming yet. BuildInfo, VersionInfo, BranchInfo, RepoInfo, dotnet/status, ...  If you have thoughts please let me know.